ocaml/utils/config.mli

```diff
@@ -233,7 +233,14 @@ val with_cmm_invariants : bool
 (** Whether the invariants checks for Cmm are enabled *)
 
 val reserved_header_bits : int
-(** How many bits of a block's header are reserved *)
+(** How many bits of a block's header are reserved. This is correct
+   regardless of whether we're in runtime 4 or runtime 5.
+
+   In runtime 5, this corresponds to the HEADER_RESERVED_BITS C preprocessor
+   macro.  In runtime 4, this corresponds to the PROFINFO_WIDTH C preprocessor
+   macro (and is controlled by the --with-profinfo and --enable-profinfo-width
```

Comment:
The part in parenthesis should be removed now, I think?

Reply:
Yes, thanks.